app/experimenter/experiments/changelog_utils.py

```diff
@@ -1,4 +1,24 @@
-from experimenter.experiments.models import ExperimentChangeLog
+from django.contrib.auth import get_user_model
+from django.conf import settings
+
+from experimenter.experiments.models import Experiment, ExperimentChangeLog
+from experimenter.experiments.serializers.entities import ChangeLogSerializer
+
+
+def update_experiment_with_change_log(experiment, changed_data):
```

Comment:
Ah I would call this `old_experiment` rather than reuse and reassign to `experiment`, could introduce wonkiness in the future to reuse variable names